scripts/SREP/infra-node-resize/script.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+# Infra-node-resize
+
+set -x
+set -e
+set -o nounset
+set -o pipefail
+
+# find hive_shard ID
+CLUSTER_ID=ocm list clusters --parameter search="external_id is '${1}'" --columns id --padding 20 --no-headers
+HIVE_ID=$(ocm get /api/clusters_mgmt/v1/clusters/${CLUSTER_ID}/provision_shard | jq -r '.hive_config.server' | cut -d '.' -f 2)
+DESIRED_SIZE=x
+
+#get current infra_size
+get_current_infra_size() {
+    ocm backplane login $HIVE_ID
+    oc get ns | grep $CLUSTERID
+    oc -n uhc-production-$CLUSTER_ID get machinepool
+    INFRA_MACHINEPOOL=$(oc -n uhc-production-$CLUSTERID  get machinepool -o json | jq -r '.items[] | select(.spec.name=="infra") | .metadata.name')
+    oc -n uhc-production-$CLUSTERID get machinepool $INFRA_MACHINEPOOL -o json | jq -r ".spec.platform"
+}
+
+# review the size
+# review_current_infra_size() {}
+
+if [[review_current_infra_size()]]; then
+    oc -n uhc-production-$CLUSTER_ID get machinepool $INFRA_MACHINEPOOL -o yaml > /tmp/cluster-infra.yaml
+fi
+
+# update the yaml file
+update_cluster-infra-yaml() {
+yq '.items[] |= (del(.metadata.creationTimestamp) | del(.metadata.finalizer) | del(.metadata.resourceVersion) | del(.metadata.generation) | del(.metadata.selfLink) | del(.metadata.uid) | del(.status)'
+yq `.metadata.name = .metadata.name | del(.metadata.name.)`
+}
+
+# silent the alerts
+silent_alerts() {
+    echo "To run this you need to have alerts script in your system"
```

Comment:
```suggestion
    echo "To run this you need to have alerts script in your system"
```
I am confused as to what part of the code handles if the script is not present in someone's system?

Reply:
Added a condition to check that...will add further to download the file if not present.